packages/web/src/components/spa/HackAUBG/GradingCriteria/grading_criteria.css

```diff
@@ -48,7 +48,7 @@
 @media only screen and (max-width: 900px) {
```

Comment:
On desktop there is more padding/margin on the top than on the bottom. Make it the same for both desktop and mobile devices